examples/plotting_tools/plot_a2c_hyperparams.py

```diff
@@ -4,7 +4,8 @@
 import matplotlib as mpl
 import numpy as np
 from matplotlib import pyplot as plt
-from plot_dqn_hyperparams import concatenate_results
+
```

Comment:
Just curious. Why the extra space?

Reply:
Those imports were broken, fixing the import the causes ruff/isort to put it seepage cause its a local import, which those tools group separately

Reply:
Interesting. Thanks for the explanation